packages/ui/Scroller/Scroller.js

```diff
@@ -53,7 +53,13 @@ const scrollerDefaultProps = {
 const Scroller = (props) => {
 	// Hooks
 
-	const scrollerProps = Object.assign({}, scrollerDefaultProps, props);
+	const scrollerProps = Object.assign({}, props);
+	for (const prop in scrollerDefaultProps) {
```

Comment:
Let's make a util function for this and use it through ui, sandstone, and agate.
Input would be props and defaultProps objects and it will return a object that has the default prop set if a prop set to `undefined`.

Reply:
I added util function